.appveyor.yml

```diff
@@ -0,0 +1,18 @@
+os: Visual Studio 2017
+
+install:
+  - appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
+  - rustup-init -y --default-toolchain stable --default-host x86_64-pc-windows-msvc
+  - set PATH=%PATH%;%USERPROFILE%\.cargo\bin
+  - set RUST_BACKTRACE=1
+  - rustc -vV
+  - cargo -vV
+  # For build Compiling jni v0.10.2 
+  - ps: Copy-Item -Path C:\Progra~1\Java\jdk1.8.0\lib\jvm.lib -Destination C:\Progra~1\Java\jdk1.8.0\jre\bin\server\jvm.lib
+
+build: false
+
+test_script:
+  - cargo build
+  - cargo test --features=backtrace
```

Comment:
Why do we need separate `cargo test` invocations here instead of a single one (`cargo test --features backtrace,invocation`)? We have single one in travis.yml.